include/gsl/gsl

```diff
@@ -109,6 +109,13 @@ private:
     T ptr_;
 };
 
+template <class T>
```

Comment:
from what I can tell, ostream seems to be enough. @neilmacintosh would you like me to change that? I made the change locally and it comes and tests fine